build.sbt

```diff
@@ -47,3 +88,31 @@ lazy val core = project
       "org.specs2" %% "specs2-core" % "4.8.3" % Test),
     packageBin in Compile := (assembly in Compile).value)
   .enablePlugins(QuasarPlugin)
+
+
+lazy val s3 = project
+  .in(file("s3"))
+  .dependsOn(core)
+  .settings(commonSettings)
+  .settings(name := "quasar-destination-avalanche-s3")
+  .settings(
+    assemblyExcludedJars in assembly := {
+      val cp = (fullClasspath in assembly).value
+      cp.filter(_.data.getName != "iijdbc.jar") // exclude everything but iijdbc.jar
+    },
+    quasarPluginName := "avalanche-s3",
+    quasarPluginQuasarVersion := managedVersions.value("precog-quasar"),
+    quasarPluginDestinationFqcn := Some("quasar.destination.avalanche.s3.AvalancheS3DestinationModule$"),
+    quasarPluginDependencies ++= Seq(
+      "io.argonaut" %% "argonaut" % "6.3.0-M2",
+      "org.slf4s" %% "slf4s-api" % "1.7.25",
+      "org.typelevel" %% "cats-core" % "2.1.0",
+      "org.tpolecat" %% "doobie-core" % DoobieVersion,
+      "org.tpolecat" %% "doobie-hikari" % DoobieVersion,
+      "com.precog" %% "async-blobstore-s3" % managedVersions.value("precog-async-blobstore"),
+      "com.precog" %% "async-blobstore-core" % managedVersions.value("precog-async-blobstore")),
```

Comment:
Why the separate dependency on `-core`? Presumably it also comes transitively via `-s3`?